android/src/main/java/com/tanguyantoine/react/MusicControlModule.java

```diff
@@ -57,6 +64,8 @@
 
     public NotificationClose notificationClose = NotificationClose.PAUSED;
 
+    public static final String CHANNEL_ID = "12min_playback_channel";
```

Comment:
Can you change the channel ID to something like "react-native-music-control"?

Reply:
Off course